docs/bonsai-ui/checkout/index.mdx

```diff
@@ -231,14 +248,17 @@ type BonsaiEvent =
   | {
       tag: 'shipping_info_entered';
       event: {
+        cartId: string;
```

Comment:
Technically this could be sent as `'empty'`, but I don't think it is practically expected, and might just lead to confusion. Unsure if worth documenting 🤔